#Camera topic here
camera_topic: "/usb_cam/image_raw"
queue_size: 1

# Set your detection topic here
detection_topic: "/darknet_ros/bounding_boxes"

# Set your output topic here
tracker_topic: "/sort_track_deep"


image_view:

  enable_opencv: true
  wait_key_delay: 1
  enable_console_output: true
